packages/remix-docs-gen/src/export.ts

```diff
@@ -1,20 +1,55 @@
 import chalk from "chalk";
 import fs from "fs-extra";
 import * as path from "path";
-import { ComputedRoute } from "./types";
+import { ComputedRoute, Route, RouteDocsAggregate } from "./types";
 import { logSuccess } from "./utils";
 
+const generateDocsForRoutes = (routes: ComputedRoute[]) => {
+  return routes.map((item) => `"${item.path}": ${item.docs},`).join("\n    ");
+};
+
 export const exportDocs = (routes: ComputedRoute[], outputPath: string) => {
-  const routeDocs = routes
-    .map((item) => `"${item.path}": ${item.docs},`)
-    .join("\n    ");
+  const routeDocsAggregate = routes.reduce<RouteDocsAggregate>(
+    (acc, route) => {
+      const doesLoaderExist = route.docs?.includes("loader: { output: "); // this definitely isn't a perfect way of checking, but it's pretty unlikely to not be the case
```

Comment:
this strategy of checking can DEFINITELY be improved, one of the ways I imagine doing that is going deeper into when these get generated and doing it there. 

It was a little tough for me to accomplish that just because I've never really worked with local filesystem in node and I have a lot to learn about how you're actually generating these types